content/en/docs/next/op-guide/supported-platform.md

```diff
@@ -8,29 +8,29 @@ description: etcd support status for common architectures & operating systems
 
 The following table lists etcd support status for common architectures and operating systems:
 
-| Architecture | Operating System | Status       | Maintainers                 |
+| Architecture | Operating System | Support Tier | Maintainers                 |
```

Comment:
- [ ] The *Support Tier* column only lists tier levels, so the column values can be just numbers. That is, in the table, I'd suggest replacing Tier-x simply by x.
- [ ] It might make more sense to sort the table rows on the *Architecture* column -- currently it seems sorted on tier (but a reader is more likely to be looking for a specific architecture than a tier).